docs/anydao/design.md

```diff
@@ -0,0 +1,32 @@
+# Design
+
+## Architecture
+The whole system for **AnyDAO MVP** is composed of 4 major components:
+
+- **Substrate based blockchain**: It stores Project, Proposal and other related data to provide finality and credibility.
+- **IPFS**: It stores less size-sensitive data such as texts, images and public voter information.
```

Comment:
Stores the actual bits of the data such as texts, images and public voter information.